mubench.reviewsite/templates/review.phtml

```diff
@@ -109,15 +109,10 @@ use MuBench\ReviewSite\Model\Experiment;
             <td>
                 <?php if($misuse->hasCode()) : ?>
                     <?php foreach($misuse->getCode() as $c): ?>
```

Comment:
Please use descriptive names for the variables, e.g., $code. I would actually prefer the method to be called getSnippets() and the variable $snippet, in this case.